BUILD

```diff
@@ -0,0 +1,38 @@
+load(
+    "@build_bazel_rules_apple//apple:macos.bzl",
+    "macos_unit_test",
+)
+
+load(
+    "@build_bazel_rules_apple//apple:ios.bzl",
+    "ios_unit_test",
+)
+
+load(
+    "@build_bazel_rules_swift//swift:swift.bzl",
+    "swift_library",
+)
+
+swift_library(
+  name = "Mapper",
+  srcs = glob(["Sources/*.swift"]),
+  module_name = "Mapper",
```

Comment:
I think this was discussed elsewhere, but why doesn't `module_name` default to `name`?

Reply:
Hmm actually since this is in the root of the repo, I guess I don't need this. If this was nested in `some/path/Mapper` the module name would otherwise be `some_path_mapper` and you'd have to import it with that.